### DISCUSSION TITLE
password reset -  ngnix proxy manager

### DISCUSSION THREAD

**Discussion by CustomerJFK** [ej52/proxmox-scripts#141] [General]
Hi 
I forgot my password of my web-ui of my ngnix proxy manger. How can I reset the password?
Would be great if anyone can help!
Thanks

**Comment by Eeems** [comment]
> Hi I forgot my password of my web-ui of my ngnix proxy manger. How can I reset the password? Would be great if anyone can help! Thanks

You'll probably want to ask on the nginx proxy manager repository, this is for a script that sets it up in an lxc container in proxmox, not for supporting the underlying tool itself.
